src/appmixer/servicenow/record/CreateRecord/component.json

```
@@ -1,53 +1,74 @@
+                    "sysparm_fields": { "type": "string" },
+                    "sysparm_view": { "type": "string" },
+                    "sysparm_query_no_domain": { "type": "boolean" }
+                },
+                "required": [
+                    "tableName", "record"
+                ]
             },
             "inspector": {
                 "inputs": {
+                    "tableName": {
+                        "type": "text",
+                        "label": "Table name",
+                        "index": 1,
+                        "tooltip": "Name of the table from which to retrieve the record."
+                    },
+                    "record": {
+                        "type": "textarea",
+                        "label": "Record",
+                        "index": 2,
+                        "tooltip": "JSON object containing field values for the new record. Example: <code>{\"email\":\"foo@bar.baz\",\"name\":\"Foo\"}</code>."
+                    },
+                    "sysparm_display_value": {
+                        "type": "toggle",
+                        "label": "Display value",
+                        "index": 3,
+                        "tooltip": "Return field display values (true), actual values (false), or both (all) (default: false)"
+                    },
+                    "sysparm_exclude_reference_link": {
+                        "type": "toggle",
+                        "label": "Exclude reference link",
+                        "index": 4,
+                        "tooltip": "True to exclude Table API links for reference fields (default: false)"
+                    },
                     "sysparm_fields": {
                         "type": "text",
                         "label": "Fields",
-                        "index": 1,
-                        "tooltip": "A comma-separated list of fields to restrict the generated inputs and limit the properties included in the response. If left empty, it will generate and return all available inputs and properties.",
-                        "group": "options"
+                        "index": 5,
+                        "tooltip": "A comma-separated list of fields to return in the response"
+                    },
+                    "sysparm_view": {
+                        "type": "text",
+                        "label": "View",
+                        "index": 6,
+                        "tooltip": "UI view for which to render the data. Determines the fields returned in the response. Valid values: desktop, mobile, both. If you also specify the sysparm_fields parameter, it takes precedent."
+                    },
+                    "sysparm_query_no_domain": {
+                        "type": "toggle",
+                        "label": "Query no domain",
+                        "index": 7,
+                        "tooltip": "True to access data across domains if authorized (default: false)"
```

Same as "sysparm_exclude_reference_link":
```suggestion
                    "sysparm_query_no_domain": {
                        "type": "toggle",
                        "label": "Query no domain",
                        "index": 7,
                        "tooltip": "True to access data across domains if authorized",
                        "defaultValue": false
```